packages/web/src/common/store/account/sagas.js

```diff
@@ -167,6 +167,7 @@ function* associateTwitterAccount(action) {
   const { uuid, profile } = action.payload
   yield waitForWrite()
   const audiusBackendInstance = yield getContext('audiusBackendInstance')
+  // TODO: ABE associate functions don't throw, so we won't catch errors here.
```

Comment:
I don't think ABE is commonly understood, took me a sec

Reply:
Oh no it's not. This was just for me to do before merging. Apologies!